charts/itsm-ng/templates/deployment.yaml

```diff
@@ -32,6 +32,14 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: 
```

Comment:
`command` overrides the entrypoint, therefore you need to restore the default workflow at the end of your custom script (using `exec` and reporting the arguments provided).